atom/browser/api/atom_api_app.cc

```diff
@@ -1162,6 +1164,33 @@ v8::Local<v8::Value> App::GetGPUFeatureStatus(v8::Isolate* isolate) {
   return mate::ConvertToV8(isolate, status ? *status : temp);
 }
 
+util::Promise* App::GetGPUInfo(v8::Isolate* isolate,
+                               const std::string& info_type) {
+  if (info_type != "available" && info_type != "complete") {
```

Comment:
Can you rename `available` to `basic` , the former seems too generic. Also the result from `CollectBasicGraphicsInfo` seems fixed, it would be best to list the available data from it in the docs explicitly, that way users would have better idea to choose among the options.